.github/workflows/bump-golang-1.18.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Why split it into three workflows instead of one matrix workflow?

Reply:
i initially did it, but I somehow found the `filename` with the go-version much appealing and easy to track that having one workflow with the matrix list. It's just a cosmetic change, but files with simpler workflows might help with the maintenance

Reply:
Edit one alternative for the matrix vs create/delete new files. 
If the matrix file contains the version and the SHA we can get rid of part of the update script and load the SHA and version from that file too.

```yaml
versions:
  - name: 1.20.0
    sha: 98579582350y183425198375098370597340597340
  - name: 1.19.0
    sha: 98579582350y183425198375098370597340597340
  - name: 1.18.0
    sha: 98579582350y183425198375098370597340597340
```

Anyways it looks like more implementation effort right now so let's live with three workflows

Reply:
That's a good approach, though it might require to change a bit more. For now as you said, let's keep it simple to support the current implementation. This might be changed by the ingest team in the future, maybe.